### PR TITLE
Update changelog for 1.36.1 hotfix

### PR DESCRIPTION
**Pull request by kburtram**
Update changelog for 1.36.1 hotfix release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Change Log
 
+## Version 1.36.1
+* Release date: April 22, 2022
+* Release status: General Availability
+## What's new in this version
+- April Hotfix addressing these issues https://github.com/microsoft/azuredatastudio/milestone/88?closed=1.
+
+| Platform																|
```
Comment (kburtram) [inline CHANGELOG.md:9]: @erpett is the plan to include links to each build now?  or only the current one?  I wasn't sure what you had in mind with the link section in this document.

Comment (erpett) [reply CHANGELOG.md:9]: The instructions I got when i took over release management is that we only include the latest links in the changelog and the fwlinks for the hotfix should be the same as yesterday's release

Comment (erpett) [reply CHANGELOG.md:9]: So I have just been moving the "old" notes to below the links and adding the new notes above it.

Comment (kburtram) [reply CHANGELOG.md:9]: thanks @erpett.  I'll update the PR to remove the links from 1.36.0 entry in that case.

**Review by erpett — APPROVED** [review]

**Pull request merged**